packages/textkit/src/run/omit.js

```diff
@@ -5,11 +5,9 @@
  * @return {Object} run without ommited attribute
  */
 const omit = (value, run) => {
-  const attributes = Object.assign({}, run.attributes);
-
-  delete attributes[value];
-
-  return Object.assign({}, run, { attributes });
+  const newAttributes = {...run.attributes};
+  delete newAttributes[value];
+  return {...run, attributes: newAttributes};
```

Comment:
So this is a bit faster on my mac.

If this proves to be faster in cloud run, too, then we should consider replacing each of the (many!) uses of `Object.assign()` with the spread operator.

I expect actual performance to vary heavily with the machine where it's running, node version, etc.